docs/api-reference/view-state-transitions.md

```diff
@@ -4,7 +4,7 @@ View state transitions provide smooth and visually appealing transitions when Vi
 
 Following fields of `viewState` can be used to achieve viewport transitions.
 
-* `transitionDuration` (Number, optional, default: 0) - Transition duration in milliseconds, default value 0, implies no transition.
+* `transitionDuration` (Number|String, optional, default: 0) - Transition duration in milliseconds, default value 0, implies no transition. It can also be set to `'auto'` where actual duration is auto calculated based on start and end viewports and is linear to the distance between them. This duration can be further customized using `speed` parameter to `FlyToInterpolator` constructor.
```

Comment:
We need to clarify that `auto` mode only works when using `FlyToInterpolator`,  add the new text in a separate line :
```
... default value 0, implies no transition.
When using `FlyToInterpolator`, it can also be set to `auto` where ...
```